documentation/advanced/tutorial-preserving-state-on-refresh.asciidoc

```diff
@@ -32,9 +32,9 @@ public class PreservedView extends VerticalLayout {
 }
 ----
 
-If the view component is defined to have a <<../routing/tutorial-router-layout#,router layout>> (via the `layout` parameter of the `@Route` annotation), the router layout is also preserved on refresh.
+If the view component is defined to have a router layout (via the `layout` parameter of the `@Route` annotation), the router layout is also preserved on refresh. See <<../routing/tutorial-router-layout#,6.3. _Router Layouts and Nested Router Targets_>> for more.
```

Comment:
Note that due to [this](https://github.com/vaadin/book-of-vaadin-tools/blob/f4ba41514002b04b54d1e78b598440f891171aec/book-sources-maven-plugin/src/main/java/com/vaadin/book/sources/ParseSources.java#L187) line in the parsing code, the link will be removed.